lib/module.js

```diff
@@ -163,24 +163,19 @@ export default function SentryModule (moduleOptions) {
   }
 
   // Enable publishing of sourcemaps
-  if (!options.disabled) {
-    this.extendBuild((config, { isClient, isModern, isDev }) => {
-      if (!options.publishRelease || isDev) {
-        return
-      }
+  if (options.publishRelease && !options.disabled && !this.options.dev) {
+    this.nuxt.hook('webpack:config', (webpackConfigs) => {
+      webpackConfigs.forEach((config) => {
```

Comment:
I generally prefer for/of loops. No unnecessary function calls then.
It's just a note though as the behavior is the same.